src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/ServerVariableConstants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.AspNetCore.Rewrite.Internal.IISUrlRewrite
```

Comment:
style-wise, do you prefer a separate file or would these be better as public constants directly on `ServerVariable`?